### ISSUE TITLE
Templates not updating on save

### ISSUE THREAD

**Issue by ru7hl355** [twigjs/twig.js#508]
Im having an issue where, when i save a template that includes other twig files, if the twig files have been updated they don't pull through to the template.

I have set Twig.cache(false) but it hasn't helped, removing the contents of this function however fixes the issue:

```
Twig.Templates.save = function(template) {
        if (template.id === undefined) {
            throw new Twig.Error("Unable to save template with no id");
        }
        Twig.Templates.registry[template.id] = template;
    };
```
this is how im rendering my templates:

```
      Twig.renderFile(paths.views + 'twig/pages/itemView.twig', context, function(err, html) {

        renderedHtml = Twig.twig({
          data: html,
          allowInlineIncludes: true,
          async: false
        });

        output = renderedHtml.render(context.component.context);
        grunt.file.write(paths.dist + folder + '/' + componentInfo.name + '.html', output, encodingUtf8);
      });
```
any guidance would be appreciated.

**Comment by olets** [comment]
Are you watching for changes? Grunt needs to know to rerun the compilation when twig files change. This is typically done with [`grunt-contrib-watch`](https://github.com/gruntjs/grunt-contrib-watch), as described in Grunt's [Sample Gruntfile](https://gruntjs.com/sample-gruntfile)

**Comment by ru7hl355** [comment]
Yes, all files bring watched. And page refreshes on change, but markup is cached. Context changed pull through fine. By removing the save function, things work as expected...

**Comment by willrowe** [comment]
Closing since this is so old. Please reopen if this is still an issue.
